runtime/src/lib.rs

```diff
@@ -964,6 +300,7 @@ sp_api::impl_runtime_apis! {
             add_benchmark!(params, batches, pallet_amendments, Amendments);
             add_benchmark!(params, batches, pallet_balances, Balances);
             add_benchmark!(params, batches, pallet_collective, TechnicalCommittee);
+            add_benchmark!(params, batches, pallet_contracts, Contracts);
```

Comment:
This wouldn't work if the `Contracts` pallet is not in the runtime

Reply:
Sorry overlooked, correction done :+1: